README.md
This is an obsolete archive of the website as it was in ~2021, before the invention of ServitorConnect 4443 and CLIv2/CLIv3, which are now the best ways to run the Repeater.

For up-to-date information on this and many other matters, please visit https://intentionrepeater.com and https://intentionrepeater.boards.net


Change_Log/Change_Log.txt
Intention Repeater Change Log (10/27/2020) by Thomas Sweet aka Anthro Teacher

4/13/2019 intentionrepeater.com domain registered.
1/11/2020 v0.3 Created. Could only go up to 1M iterations per second and was very basic.
8/6/2020 v4.0 Added Heart/Mind Coherence and Clearing Energetic Interference options.
9/6/2020 v5.0 Added Gut to Heart/Mind Coherence option. Added "OM" the highest vibration that made a big difference. Automated to allow running without user input from terminal.
9/7/2020 v5.1 Added Akashic Records option. They provide a greater gamut of emotions such as infinitely subtle variations on Joy. However, may also be very expansive and hard to focus down.
9/8/2020 v5.2 Changed to use ORGONE AETHER RESONATORS and ORGONE BUBBLES.
9/8/2020 v5.3 Added INFINITE ENERGY and INTELLIGENT ENERGY and LOGOS and HR 6819. This helps focus down the Intelligent Infinity. LOGOS really softens the energy and makes it more integrating. Very nice. LOGOS is another name for our galaxy's central black hole/consciousness. HR 6819 is the closest black hole system to Earth and has been relevant in working with the Repeater.
9/9/2020 v5.4 ALL AVAILABLE BENEFICIAL ENERGY GRIDS OF EARTH/GAIA FOCUSED THROUGH CRYSTAL GRID OF EARTH/GAIA. Added NODES AND NULLS OF EARTH/GAIA to work with the grids. Also added 963HZ GOD FREQUENCY. Added SIRIUS B the white dwarf star, since that was what I used during my development to increase my energy density. It makes the energy denser and smoother.
9/11/2020 v6.0 Added check (not really a timer, more like a sentinel) to stop running at a certain time. Usage: python3 intention_repeater.py YYY8 "HH:MM:SS" "Intention". Use "00:00:00" if running until manually stopped.
9/12/2020 v6.1 Added ONE INFINITE CREATOR and FULLY OPTIMIZE THE ENERGY.
9/13/2020 v6.2 Added option for SUPERCOOLED MOST PERFECTLY BALANCED, PURIST AND MOST POWERFUL QUASAR. Stronger and smoother, but can be draining. Also added GUIDED BY HIGHER SELF to help with orchestrating it. The Quasar one is best if intending to manifest something solid (though it can't quite do that yet), rather than using it for a person.
9/17/2020 v6.3 CANCEL DESTRUCTIVE OR FEARFUL INTENTIONS. SEND ALL SPECIFIED INTENTIONS, AFFIRMATIONS, AND/OR DESIRED MANIFESTATIONS, OR BETTER. PLEASE HELP USER TO RAISE THEIR VIBRATION TO THE LEVEL REQUIRED TO MAKE THEIR SPECIFIED INTENTIONS, AFFIRMATIONS, AND/OR DESIRES MANIFEST. IF THE USER IS UNABLE TO ACHIEVE THE NECESSARY VIBRATION TO MANIFEST THEIR SPECIFIED INTENTIONS, AFFIRMATIONS, AND/OR DESIRES, BUT THERE ARE BEINGS WHO COULD AND THEY ARE WILLING TO ASSIST IN MANIFESTING THE SPECIFIED INTENTIONS, AFFIRMATIONS, AND/OR DESIRES, PLEASE ENLIST THEIR AID.
9/29/2020 v6.3 still. Just changed some comments.
10/4/2020 v6.4 Formatted code to clean it up.
10/27/2020 v7.0 Simplified to: process_energy_statement = ' REGULATE AND INTEGRATE. IT IS DONE. OM.'
	and eliminated all options but the intensity and intent.

test/index.php
<HEAD>
   <title>
      Sweet Intention Repeater
   </title>
   <meta property="description" content = "Free, Open-Source Intention Repeater that repeats your intention up to 1+ million times per second to make things happen." />
   <meta property="og:title" content="Sweet Intention Repeater">
   <meta property="og:image" content="http://www.intentionrepeater.com/images/Sweet_Intention_Repeater_Image_Main.jpg">
   <meta property="og:image:type" content="image/jpeg">
   <meta property="og:description" content = "Free, Open-Source Intention Repeater that repeats your intention up to 1+ million times per second to make things happen." />
   <meta property="og:url" content="https://www.intentionrepeater.com">
   <meta property="og:type" content="website">
   <meta name="twitter:title" content="Sweet Intention Repeater">
   <meta name="twitter:description" content="Free, Open-Source Intention Repeater that repeats your intention up to 1+ million times per second to make things happen.">
   <meta name="twitter:image" content="http://www.intentionrepeater.com/images/Sweet_Intention_Repeater_Image_Main.jpg">
   <meta name="twitter:card" content="summary_large_image">
   <script src="https://kit.fontawesome.com/003fd16dd7.js" integrity="sha384-eoEIP4kL867FWM1D0Gem6jxjxMlsD5jiBy1lo9hB45B9e6PLQZ96UxgVowiBgMxx" crossorigin="anonymous"></script>
   <link rel="stylesheet" href="/css/styles.css">
</HEAD>
<BODY>
   <?php
      $servername = "localhost";
      $username = "furries1_IRuser";
      $password = "c3Fou,^fWQrig_JlU)";
      $dbname = "furries1_intentionrepeatercounter";
      
      // Create connection
      $conn = new mysqli($servername, $username, $password, $dbname);
      // Check connection
      if ($conn->connect_error) {
          die("Connection failed: " . $conn->connect_error);
      } 
      
      $sql = "INSERT INTO ip_table (ip_address) VALUES ('1')";
      
      if ($conn->query($sql) === TRUE) {
          echo "";
      } else {
          echo "Error: " . $sql . "<br>" . $conn->error;
      }
      
      $sql = "UPDATE table_count set num_records = num_records + 1";
      
      if ($conn->query($sql) === TRUE) {
          echo "";
      } else {
          echo "Error: " . $sql . "<br>" . $conn->error;
      }
      
      $conn->close();
      ?>
   <center>
   <img src="/images/SweetIntentionRepeater.png" alt='Sweet Intention Repeater header'>
   <BR><BR>
   <p class='boldtext'>Intention: <input type="text" id='intention' size="100">
      <input type="button" name="btn" id='btn' value="Start" onclick="to_start()";><BR>
   <div id="n1" class="boldtext"> </div>
   </p>
   <script type="text/javascript" src="/js/Repeater.js"></script>
   <center>
      <BR><BR>
      <p class='normaltext'>
         A powerful free, open-source tool for manifestation, blessing, energy clearing, healing, balancing and more.<BR>
         Repeat your intention up to 1+ Million times per second to make things happen (with Python version).<BR>
         Tap Intelligent Infinity/Magick; powered by a black hole (Servitor) 20 years / 2000+ hours in its co-creation.<BR><BR>
         <a href='healings.html'>Click Here for my Healing Services</a><BR>
         Please contact me for my Repeater consulting services for your product.<BR>
         Or to have me charge up an object for you as a manifestor.<BR><BR>
         <b>My Book:</b><BR><BR>
         </font>
         <a href='https://www.thewarlockname.com' target='_blank'><img src="/images/TWN_Cover.jpg" alt='The Warlock Name cover'></a>
      </p>
      <?php
         $servername = "localhost";
         $username = "furries1_IRuser";
         $password = "c3Fou,^fWQrig_JlU)";
         $dbname = "furries1_intentionrepeatercounter";
         
         $link = mysqli_connect($servername, $username, $password, $dbname);
         
         /* check connection */
         if (mysqli_connect_errno()) {
             printf("Connect failed: %s\n", mysqli_connect_error());
             exit();
         }
         
         $result=mysqli_query($link,"SELECT num_records as total from table_count");
         $data=mysqli_fetch_assoc($result);
         echo "<BR><BR><b><font color='white' size='+1'>Total Page Loads: " . $data['total'] . "</font></b>";
         
             /* close result set */
             mysqli_free_result($result);
         
         /* close connection */
         mysqli_close($link);
         ?>
      <BR><BR>
      <p class='normaltext'>
      <table>
         <tr>
            <td>
               <b>
               <font size="+1" color="white">
               7/16/2020 Radio Show<BR>
               Life in the Hologram:<BR>
               </font>
               </b>
            </td>
            <td>
               <b>
               <font size="+1" color="white">
               08/18/2020 Radio Show<BR>
               Radioality:<BR>
               </font>
               </b>
            </td>
         </tr>
         <tr>
            <td>
               <audio controls>
                  <source src="/mp3/LifeInTheHologram.mp3" type="audio/mpeg">
                  Your browser does not support the audio element.
               </audio>
            </td>
            <td>
               <audio controls>
                  <source src="/mp3/Radioality_Talk.m4a" type="audio/x-m4a">
                  Your browser does not support the audio element.
               </audio>
            </td>
         </tr>
      </table>
      </p>
      <BR><BR>
      <table>
         <tr>
            <td>
               <a href="directions.html">Directions</a><BR><BR>
            </td>
            <td>
               <a href="./Python/intention_repeater.py" target="_blank">Python3 Source Code</a><BR><BR>
            </td>
            <td>
               <a href="https://choosealicense.com/licenses/gpl-3.0/" target="_blank">Code License</a><BR><BR>
            </td>
            <td>
               <a href="origins.html">Origins of this Program</a><BR><BR>
            </td>
         </tr>
         <tr>
            <td>
               <a href="freeimages.html">Repeater Images</a><BR><BR>
            </td>
            <td>
               <a href="chakras.html">Chakra Meditations</a><BR><BR>
            </td>
            <td valign='top'>
               <script type="text/javascript">
                  <!-- Begin
                  user = "healing";
                  domain = "intentionrepeater.com";
                  document.write('<a href=\"mailto:' + user + '@' + domain + '\">Contact Us</a>');
                  // End -->
               </script>
               <noscript><img src='/images/Email.png' alt='Contact Us'></noscript>
            </td>
            <td>
               <a href="news.html">News</a><BR><BR>
            </td>
         </tr>
         <tr>
            <td>
               <a href="faq.html">FAQ</a><BR><BR>
            </td>
            <td>
               <a href="links.html">Links</a><BR><BR>
            </td>
            <td>
               <a href="usage.html">Python Usage</a><BR><BR>
            </td>
            <td><a href="https://forums.intentionrepeater.com/" target='_blank'>Forums</a><BR><BR></td>
         </tr>
         <tr>
            <td><a href="/Change_Log/Change_Log.txt" target='_blank'>Change Log</a><BR><BR></td>
            <td colspan='3'><a href="https://enlightenedstates.com/2017/04/07/servitor-just-powerful-spiritual-tool/" target='_blank'>Servitor Info</a><BR><BR></td>
         </tr>
         <tr>
            <td style="text-align:center"><a href="https://www.facebook.com/IntentionRepeater" target='_blank' class='fontawesome'><i class="fab fa-facebook"></i></a></td>
            <td style="text-align:center"><a href="https://twitter.com/TeacherAnthro" target='_blank' class='fontawesome'><i class="fab fa-twitter"></i></a></td>
            <td style="text-align:center"><a href="https://www.instagram.com/anthroteacher/" target='_blank' class='fontawesome'><i class="fab fa-instagram"></i></a></td>
            <td style="text-align:center"><a href="https://www.youtube.com/c/AnthroTeacher/" target='_blank' class='fontawesome'><i class="fab fa-youtube"></i></a></td>
         </tr>
         <tr>
            <td style="text-align:center"><a href="https://www.patreon.com/AnthroTeacher" target='_blank' class='fontawesome'><i class="fab fa-patreon"></i></a></td>
            <td style="text-align:center"><a href="https://independent.academia.edu/ThomasSweet2" target='_blank' class='fontawesome'><i class="fas fa-book"></i></i></a></td>
            <td style="text-align:center"><a href="/Python/intention_repeater.py" target='_blank' class='fontawesome'><i class="fab fa-python"></i></a></td>
            <td style="text-align:center"><a href="https://www.venmo.com/AnthroTeacher" target='_blank' class='fontawesome'><i class="fas fa-life-ring"></i></a></td>
         </tr>
      </table>
      <BR>
      <img src="/images/Repeater_Review.png" alt='Intention Repeater review 1'>
      <BR><BR>
      <img src="/images/Repeater_Review2.png" alt='Intention Repeater review 2'>
      <BR><BR>
      <img src="/images/Repeater_Review4.png" alt='Intention Repeater review 4'>
      <BR><BR>
      <img src="/images/Repeater_Review5.png" alt='Intention Repeater review 5'>
      <BR><BR>
      <img src="/images/Repeater_Review3.png" alt='Intention Repeater review 3'>
      <BR><BR>
      <a href='https://www.facebook.com/TheWarlockName/' target="_blank"><img src="/images/TWN_Quote.png" alt='A quote from The Warlock Name'></a>
      <BR><BR>
      <iframe width="560" height="315" src="https://www.youtube.com/embed/W7qFZgsWRww?enablejsapi=1&origin=https://www.intentionrepeater.com/index.php" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      &nbsp;
      <iframe width="560" height="315" src="https://www.youtube.com/embed/RxZ_TFrviIA?enablejsapi=1&origin=https://www.intentionrepeater.com/index.php" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <BR><BR>
      <a href='https://www.venmo.com/AnthroTeacher' target="_blank"><img src="/images/Venmo.png" alt='Donate with Venmo'></a>
   </center>
</BODY>


test/js/Repeater.js
var h=0;
var m=0;
var s=0;
var i=0;
var repeats=0;
var intent=[""];

function to_start(){

switch(document.getElementById('btn').value)
{
case  'Stop':
window.clearInterval(tm); // stop the timer 
document.getElementById('n1').innerHTML = "Intention Repeated " + repeats + " times.";
document.getElementById('btn').value='Start';
break;
case  'Start':
tm=window.setInterval('disp()',1000);
document.getElementById('btn').value='Stop';
break;
}
}

function disp(){
// Format the output by adding 0 if it is single digit //
if(s<10){var s1='0' + s;}
else{var s1=s;}
if(m<10){var m1='0' + m;}
else{var m1=m;}
if(h<10){var h1='0' + h;}
else{var h1=h;}
// Display the output //
str= h1 + ':' + m1 +':' + s1 ;
document.getElementById('n1').innerHTML=str+' '+document.getElementById('intention').value;

for (i = 0; i < 10000; i++) { 
	intent.push(document.getElementById('intention').value+" ONE INFINITE CREATOR. INTELLIGENT INFINITY. INFINITE ENERGY. INTELLIGENT ENERGY. LOGOS. HR 6819. BY GRACE. IN COOPERATION WITH FATHER GOD AND MOTHER GODDESS AND SOURCE. PURE ADAMANTINE PARTICLES OF LOVE/LIGHT. IN THE HIGHEST AND GREATEST GOOD OF ALL. REQUESTING AID FROM ALL BENEVOLENT BEINGS WHO ARE WILLING TO ASSIST. METATRON''S CUBE. 0010110. GREAT CENTRAL SUN. SIRIUS A. SIRIUS B. SOL. ALL AVAILABLE BENEFICIAL ENERGY GRIDS OF EARTH/GAIA FOCUSED THROUGH CRYSTAL GRID OF EARTH/GAIA. NODES AND NULLS OF EARTH/GAIA. CREATE STABILIZATION FIELD. CREATE ZONE OF MANIFESTATION. ALL AVAILABLE ORGONE AETHER RESONATORS. ALL AVAILABLE ORGONE BUBBLES. USE EVERY AVAILABLE RESOURCE (RESPECTING FREE WILL). MANIFEST ASAP AT HIGHEST DENSITY POSSIBLE INTO BEST DENSITY FOR USER. CREATE STRUCTURE. 963HZ GOD FREQUENCY. 432HZ MANIFESTATION. CANCEL NEGATIVE INTENTIONS. PURIFY THE ENERGY. CLEAR THE BLOCKAGES. REGULATE AND BALANCE THE ENERGY. USE THE MOST EFFECTIVE PATH IN THE MOST EFFICIENT WAY. FULLY OPTIMIZE THE ENERGY. INTEGRATE THE ENERGY. PROCESS THE CHANGES. GUIDED BY MY HIGHER SELF. GROUNDED TO GAIA, CONNECTED TO SOURCE, INTEGRATING BOTH WITHIN THE SACRED HEART. IT IS DONE. SO SHALL IT BE. NOW RETURN A PORTION OF THE LOVE/LIGHT RECEIVED AND ACTIVATED BACK INTO THE HIGHER REALMS OF CREATION. I LOVE YOU AND THANK YOU. OM");
	repeats+=1;
}
intent.length=0;

// Calculate the stop watch // 
if(s<59){ 
s=s+1;
}else{
s=0;
m=m+1;
if(m==60){
m=0;
h=h+1;
} // end if  m ==60
}// end if else s < 59
// end of calculation for next display

}


Python/intention_repeater.py
#!/usr/bin/python
# -*- coding: utf-8 -*-

# Intention Repeater created by Thomas Sweet
# Updated 10/27/2020 v7.0
# Requires Python v3.5.3 or greater
# Repeats intention up to 1M+ times per second
# Usage instructions: https://www.intentionrepeater.com/usage.html
# Run using Windows: intention_repeater.py
# Run using Linux/MacOS: python3 intention_repeater.py
# Automated Example Linux/MacOS: python3 intention_repeater.py 8 "HH:MM:SS" "Intentions/Filename with Intentions"
# Automated Example Windows: intention_repeater.py 8 "HH:MM:SS" "Intentions/Filename with Intentions"
# Intention Repeater is powered by a Servitor (20 Years / 2000+ hours in the making)
# Servitor Info: https://web.archive.org/web/20200915191532/https://enlightenedstates.com/2017/04/07/servitor-just-powerful-spiritual-tool/
# Website: https://www.intentionrepeater.com/
# Forum: https://forums.intentionrepeater.com/
# Licensed under GNU General Public License v3.0
# This means you can modify, redistribute and even sell your own modified software, as long as it's open source too and released under this same license
# https://choosealicense.com/licenses/gpl-3.0/

import time, sys, sched
from math import ceil, log10

def human_format(num):
    num = float('{:.3g}'.format(num))
    magnitude = 0
    while abs(num) >= 1000:
        magnitude += 1
        num /= 1000.0
    return '{}{}'.format('{:f}'.format(num).rstrip('0').rstrip('.'), [
        '',
        'K',
        'M',
        'B',
        'T',
        'Q',
        ][magnitude])


print("Intention Repeater v7.0 software created by Thomas Sweet.\n")
print("This software comes with no guarantees or warranty of any kind.\n")

args = list(sys.argv)

try:
    intensityparam = args[1]
    runtimeparam = args[2]
    intentparam = args[3]
except:
    intensityparam = ''
    runtimeparam = ''
    intentparam = ''

list_write = []

intention = ''
intensity = ''

if intentparam == '':
    while intention == '':
        intention = input('What is your intent?: ')
else:
    intention = intentparam

# We want to cancel negative intentions. Regulate energy so that it doesn't get overpowering.
# Choose the most effective and efficient path. And conclude with it is done on each iteration.

process_energy_statement = ' REGULATE AND INTEGRATE OM IT IS DONE'

intentionval = intention + ' ' + process_energy_statement

benchmark = 0
start_time = float(time.time())

# See how many iterations the processor can run in one second.

while float(time.time()) - start_time < 1.0:
    benchmark += 1
    list_write.append(intentionval)

list_write.clear()

# Determine how many 0's the benchmark number would equate to. Like 100k or a million or so iterations in a second.
# Add 1 for any partial million or 100k left over.

maxintensitylevel = ceil(log10(benchmark)) + 1

if intensityparam == '':
    while intensity is not int:
        try:
            intensity = int(input('Intensity [1-'
                            + str(maxintensitylevel) + ']: '))
            break
        except ValueError:
            print("Please enter a valid intensity value.")
else:
    intensity = int(intensityparam)

if intensity > maxintensitylevel:
    intensity = maxintensitylevel

if intensity < 1:
    intensity = 1

if intensity == maxintensitylevel:
    maxintensity = benchmark
else:
    maxintensity = 10 ** (intensity - 1)

num_writes = 0

print("Press CTRL-C to stop running.\n")

# Calculate how long it takes to run the number of iterations for level of intensity chosen.

start_time = float(time.time())
for d in range(maxintensity):
    list_write.append(intentionval)

# If run the last second before midnight, benchmark again.

if float(time.time() - start_time) < 0:
    list_write.clear()
    for d in range(maxintensity):
        list_write.append(intentionval)

sleeptime = 1.0 - float(time.time() - start_time)

list_write.clear()

# We write to memory a certain number of times to repeat the intention

try:
    while True:
        for d in range(maxintensity):
            list_write.append(intentionval)
            num_writes += 1
            if num_writes % maxintensity == 0:
                sys.stdout.write('  ' + time.strftime('%H:%M:%S',
                                 time.gmtime(int(num_writes
                                 / maxintensity))) + ' ['
                                 + human_format(num_writes) + '] '
                                 + intention + '   \r')
                sys.stdout.flush()
                list_write.clear()
                if runtimeparam == time.strftime('%H:%M:%S',
                        time.gmtime(int(num_writes / maxintensity))):
                    print("\nIntention repeated " \
                        + human_format(num_writes) \
                        + " times. IT IS DONE.")
                    quit()
        if sleeptime > 0:
            time.sleep(sleeptime)
except KeyboardInterrupt:

    pass

sys.stdout.write('\nIntention repeated ' + human_format(num_writes) + ' times. IT IS DONE.')

list_write.clear()

index2.php
<HEAD>
   <title>
      Sweet Intention Repeater
   </title>
   <meta property="description" content = "Free, Open-Source Intention Repeater that repeats your intention up to 1+ million times per second to make things happen." />
   <meta property="og:title" content="Sweet Intention Repeater">
   <meta property="og:image" content="http://www.intentionrepeater.com/images/Sweet_Intention_Repeater_Image_Main.jpg">
   <meta property="og:image:type" content="image/jpeg">
   <meta property="og:description" content = "Free, Open-Source Intention Repeater that repeats your intention up to 1+ million times per second to make things happen." />
   <meta property="og:url" content="https://www.intentionrepeater.com">
   <meta property="og:type" content="website">
   <meta name="twitter:title" content="Sweet Intention Repeater">
   <meta name="twitter:description" content="Free, Open-Source Intention Repeater that repeats your intention up to 1+ million times per second to make things happen.">
   <meta name="twitter:image" content="http://www.intentionrepeater.com/images/Sweet_Intention_Repeater_Image_Main.jpg">
   <meta name="twitter:card" content="summary_large_image">
   <link rel="stylesheet" href="./fontawesome6/css/all.css">
   <link rel="stylesheet" href="./css/styles.css">
</HEAD>
<BODY>
   <?php
      $servername = "localhost";
      $username = "IR_RufusAumaroo";
      $password = "+B^g2Y&b;^A}/VBm";
      $dbname = "intentionrepeater";
      
      // Create connection
      $conn = new mysqli($servername, $username, $password, $dbname);
      // Check connection
      if ($conn->connect_error) {
          die("Connection failed: " . $conn->connect_error);
      } 
      
      $sql = "INSERT INTO ip_table (ip_address, tx_user_agent) VALUES ('" . $_SERVER['REMOTE_ADDR'] . "', '" . $_SERVER['HTTP_USER_AGENT'] . "')";
      str_replace(array("'", "\"", "&quot;"), "", htmlspecialchars($sql));	  
	  
      if ($conn->query($sql) === TRUE) {
          echo "";
      } else {
          echo "Error: " . $sql . "<br>" . $conn->error;
      }
      
      $sql = "UPDATE table_count set num_records = num_records + 1";
      
      if ($conn->query($sql) === TRUE) {
          echo "";
      } else {
          echo "Error: " . $sql . "<br>" . $conn->error;
      }
      
      $conn->close();
      ?>
   <center>
   <img src="./images/SweetIntentionRepeater.png" alt='Sweet Intention Repeater header'>
   <BR><BR>
   <p class='boldtext'>Intention: <input type="text" id='intention' size="100">
      <input type="button" name="btn" id='btn' value="Start" onclick="to_start()";><BR>
   <div id="n1" class="boldtext"> </div>
   </p>
   <script type="text/javascript" src="./js/Repeater.js"></script>
   <center>
      <BR><BR>
      <p class='normaltext'>
         A powerful free, open-source tool for manifestation, blessing, energy clearing, healing, balancing and more.<BR>
         Repeat your intention over a million million times per second to make things happen (with C++ MAX version).<BR>
         Tap Intelligent Infinity/Magick; powered by a black hole (HR6819 Servitor) 20 years / 2000+ hours in its co-creation.
      <BR>

      <?php
         $servername = "localhost";
         $username = "IR_RufusAumaroo";
         $password = "+B^g2Y&b;^A}/VBm";
         $dbname = "intentionrepeater";
         
         $link = mysqli_connect($servername, $username, $password, $dbname);
         
         /* check connection */
         if (mysqli_connect_errno()) {
             printf("Connect failed: %s\n", mysqli_connect_error());
             exit();
         }
         
         $result=mysqli_query($link,"SELECT num_records as total from table_count");
         $data=mysqli_fetch_assoc($result);
         echo "<BR><BR><b><font color='white' size='+1'>Total Page Loads: " . $data['total'] . "</font></b>";
         
             /* close result set */
             mysqli_free_result($result);
         
         /* close connection */
         mysqli_close($link);
         ?>
<BR><BR>

<a href='https://github.com/tsweet77/repeater-max' target='_blank'><img src='./images/sourcecode_button.png'></a>
&nbsp;
<a href='https://intentionrepeater.boards.net' target='_blank'><img src='./images/forum_button.png'></a>
<BR><BR>

<iframe width="560" height="315" src="https://www.youtube.com/embed/Ctr-CTcuB8I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
<BR><BR>
<iframe width="560" height="315" src="https://www.youtube.com/embed/n3y2orlziuw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
<BR><BR>
<iframe width="560" height="315" src="https://www.youtube.com/embed/oIyCqFBxwWI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
<BR><BR>
         <a href='healings.html'><img src='./images/healing_services_button.png'></a><BR><BR>
         <b>My Book:</b><BR><BR>
         </font>
         <a href='https://www.thewarlockname.com' target='_blank'><img src="./images/TWN_Cover.jpg" alt='The Warlock Name cover'></a>
      </p>
      <BR><BR>
	  <a href='https://forums.intentionrepeater.com' target='_blank'><img src='./images/intentionrepeaterforum_button.png'></a>
	  <BR><BR>
      <p class='normaltext'>
      <table>
         <tr>
            <td>
               <b>
               <font size="+1" color="white">
               7/16/2020 Radio Show<BR>
               Life in the Hologram:<BR>
               </font>
               </b>
            </td>
            <td>
               <b>
               <font size="+1" color="white">
               08/18/2020 Radio Show<BR>
               Radioality:<BR>
               </font>
               </b>
            </td>
         </tr>
         <tr>
            <td>
               <audio controls>
                  <source src="./mp3/LifeInTheHologram.mp3" type="audio/mpeg">
                  Your browser does not support the audio element.
               </audio>
            </td>
            <td>
               <audio controls>
                  <source src="./mp3/Radioality_Talk.m4a" type="audio/x-m4a">
                  Your browser does not support the audio element.
               </audio>
            </td>
         </tr>
      </table>
      </p>
      <BR><BR>
      <table>
         <tr>
            <td>
               <a href="directions.html">Directions</a><BR><BR>
            </td>
            <td>
               <a href="versions.html">DOWNLOAD</a><BR><BR>
            </td>
            <td>
               <a href="https://choosealicense.com/licenses/gpl-3.0/" target="_blank">Code License</a><BR><BR>
            </td>
            <td>
               <a href="origins.html">Origins of this Program</a><BR><BR>
            </td>
         </tr>
         <tr>
            <td>
               <a href="freeimages.html">Repeater Images</a><BR><BR>
            </td>
            <td>
               <a href="chakras.html">Chakra Meditations</a><BR><BR>
            </td>
            <td valign='top'>
               <script type="text/javascript">
                  <!-- Begin
                  user = "healing";
                  domain = "intentionrepeater.com";
                  document.write('<a href=\"mailto:' + user + '@' + domain + '\">Contact Us</a>');
                  // End -->
               </script>
               <noscript><img src='./images/Email.png' alt='Contact Us'></noscript>
            </td>
            <td>
               <a href="news.html">News</a><BR><BR>
            </td>
         </tr>
         <tr>
            <td>
               <a href="faq.html">FAQ</a><BR><BR>
            </td>
            <td>
               <a href="links.html">Links</a><BR><BR>
            </td>
            <td>
               <a href="usage.html">Python Usage</a><BR><BR>
            </td>
            <td><a href="https://forums.intentionrepeater.com/" target='_blank'>Forums</a><BR><BR></td>
         </tr>
         <tr>
            <td><a href="./Change_Log/Change_Log.txt" target='_blank'>Change Log</a><BR><BR></td>
            <td><a href="https://enlightenedstates.com/2017/04/07/servitor-just-powerful-spiritual-tool/" target='_blank'>Servitor Info</a><BR><BR></td>
			<td><a href="repeater_max.html">Repeater MAX</a><BR><BR></td>
			<td><a href="repeater_wav.html">Repeater WAV</a><BR><BR></td>
		 </tr>
		 <tr>
			<td align="center"><a href="repeater_spiritual_chat.html">Spiritual Chat</a></td>
			<td align="center"><a href="repeater_wav_converter.html">WAV Converter</a></td>
			<td align="center"><a href="repeater_max_cuda.html">MAX CUDA</a></td>
         <td align="center"><a href="chambers.html">Healing Chambers</a></td>
       </tr>
      </table>
	  <BR>
      <table cellpadding='5' border='0'>
         <tr>
            <td><a href="https://www.facebook.com/IntentionRepeater" target='_blank' class='fontawesome'><i class="fab fa-facebook"></i></a></td>
            <td><a href="https://twitter.com/TeacherAnthro" target='_blank' class='fontawesome'><i class="fab fa-twitter"></i></a></td>
            <td><a href="https://www.instagram.com/anthroteacher/" target='_blank' class='fontawesome'><i class="fab fa-instagram"></i></a></td>
            <td><a href="https://www.youtube.com/c/AnthroTeacher/" target='_blank' class='fontawesome'><i class="fab fa-youtube"></i></a></td>
         </tr>
         <tr>
            <td><a href="https://www.patreon.com/AnthroTeacher" target='_blank' class='fontawesome'><i class="fab fa-patreon"></i></a></td>
            <td><a href="https://independent.academia.edu/ThomasSweet2" target='_blank' class='fontawesome'><i class="fas fa-book"></i></i></a></td>
            <td><a href="https://github.com/tsweet77" target='_blank' class='fontawesome'><i class="fab fa-github"></i></a></td>
            <td><a href="https://forums.intentionrepeater.com/" target='_blank' class='fontawesome'><i class="fab fa-discourse"></i></a></td>
         </tr>
      </table>
      <BR>
      <img src="./images/Repeater_Review.png" alt='Intention Repeater review 1'>
      <BR><BR>
      <img src="./images/Repeater_Review2.png" alt='Intention Repeater review 2'>
      <BR><BR>
      <img src="./images/Repeater_Review4.png" alt='Intention Repeater review 4'>
      <BR><BR>
      <img src="./images/Repeater_Review5.png" alt='Intention Repeater review 5'>
      <BR><BR>
      <img src="./images/Repeater_Review3.png" alt='Intention Repeater review 3'>
      <BR><BR>
      <a href='https://www.facebook.com/TheWarlockName/' target="_blank"><img src="./images/TWN_Quote.png" alt='A quote from The Warlock Name'></a>
      <BR><BR>
      <iframe width="560" height="315" src="https://www.youtube.com/embed/3lfnR7OhZY8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      &nbsp;
      <iframe width="560" height="315" src="https://www.youtube.com/embed/ZHxE-f5WXqc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <BR><BR>
	  <a href='https://paypal.me/intentionrepeater?locale.x=en_US' target='_blank'><img src='./images/PayPal.png'></a>
	  <BR><BR>
	  <a href='https://ko-fi.com/G2G21OV75' target='_blank'><img height='66' style='border:0px;height:66px;' src='https://cdn.ko-fi.com/cdn/kofi2.png?v=2' border='0' alt='Buy Me a Coffee at ko-fi.com' /></a>
	  <BR><BR>
      <a href='https://www.venmo.com/AnthroTeacher' target="_blank"><img src="./images/Venmo.png" alt='Donate with Venmo'></a>
   </center>
</BODY>
